string licensePlate
---
bool success
string message
Vehicle vehicle